orchestra/api_urls.py

```diff
@@ -1,7 +1,15 @@
 from django.conf.urls import patterns
 from django.conf.urls import url
+from orchestra.views import assign_task_api
```

Comment:
these functions should likely not all end in `_api`.  instead, consider putting them in their own set of project management views (e.g., `project_management_api_views.py`)